assets/csidriveroperators/openstack-cinder/hypershift/mgmt/generated/rbac.authorization.k8s.io_v1_role_openstack-cinder-csi-driver-operator-role.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openstack-cinder-csi-driver-operator-role
+  namespace: ${CONTROLPLANE_NAMESPACE}
+rules:
```

Comment:
Looks like this rule is missing https://github.com/openshift/cluster-storage-operator/blob/cab5822f7f768eba36473c684d903143ebeb5ea0/assets/csidriveroperators/azure-file/hypershift/mgmt/generated/rbac.authorization.k8s.io_v1_role_azure-file-csi-driver-operator-role.yaml#L75-L85

Reply:
Okay, per guidance from @gnufied I've moved this from the `ClusterRole` (which isn't present in the HCP case) to the `Role`.